### ISSUE TITLE
Broken Build: Project file 'snapcraft.yaml' not found

### ISSUE THREAD

**Issue by jimafisk** [plentico/plenti#317]
Broken build:
https://github.com/plentico/plenti/actions/runs/8883147263/job/24389561774

Error message:
```
release failed after 1m17s
error=failed to generate snap package: exit status 66: Project file 'snapcraft.yaml' not found in '/home/runner/work/plenti/plenti'.
Recommended resolution: Ensure the project file exists.
Full execution log: '/home/runner/.local/state/snapcraft/log/snapcraft-20240429-180224.885940.log'

Error: The process '/opt/hostedtoolcache/goreleaser-action/1.25.1/x64/goreleaser' failed with exit code 1
```

Context:
We just enabled a multi-step build process in order to install a root CA certificate to fix TLS errors in CI (https://github.com/plentico/plenti/issues/316). 

Problem:
There seems to be a race condition: https://github.com/goreleaser/goreleaser/issues/1715

**Comment by jimafisk** [comment]
Seems like other folks are having the same issue: https://github.com/goreleaser/goreleaser/issues/4816

**Comment by jimafisk** [comment]
I was having trouble installing a specific, older version of snapcraft, so the easiest thing to do was just to remove snap support for now.

Once there's a fix, we can add snap support back in: https://github.com/canonical/snapcraft/issues/4769

**Comment by jimafisk** [comment]
This was fixed in snapcraft 8.2.6 (https://github.com/canonical/snapcraft/issues/4769#issuecomment-2102820076).

I've added snapcraft support back in and it should be available in [v0.6.46](https://github.com/plentico/plenti/releases/tag/v0.6.46)
